src/Migrate/Migrate.Autorest/custom/New-AzMigrateServerReplication.ps1

```diff
@@ -539,6 +557,17 @@ public static int hashForArtifact(String artifact)
         $ProviderSpecificDetails.InstanceType = 'VMwareCbt'
         $ProviderSpecificDetails.LicenseType = $LicenseType
         $ProviderSpecificDetails.PerformAutoResync = $PerformAutoResync
+        if ($HasTargetVMSecureBootEnabled) {
+            $ProviderSpecificDetails.TargetVMSecurityProfileIsTargetVmsecureBootEnabled = $TargetVMSecureBootEnabled
+        }
+
+        if ($HasTargetSecurityType -and $TargetSecurityType -ne "Standard") {
```

Comment:
What if SecureBoot is not passed and SecurityType is TrustedLaunch?
Do we need to keep default value of SecureBoot as True or throw error asking user to provide SecureBoot input? Check with PM.